types/plotly.js-dist-min/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for plotly.js-dist-min 2.3
+// Project: https://github.com/plotly/plotly.js#readme
+// Definitions by: André Farzat <https://github.com/andrefarzat>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Forked from: plotly.js-basic-dist by Florian Dreier <https://github.com/DreierF>
+
+import * as Plotly from 'plotly.js';
+
+export default Plotly;
```

Comment:
Plotly appears not to have a default export, so this should probably be `export = Plotly` as in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/8d85752375d30df9c82a94af9525764581e28ca3/types/plotly.js-basic-dist/index.d.ts#L8.

Reply:
Fixed 😺